### PR TITLE
Nirs branch - learning

### PR DESCRIPTION
**Pull request by shalmonir**
Hi!
I assigned you as CR (code reviewer) and you invited to play along and leave me comments.
whatever you do, dont merge it (its not part of the SW code)

### CODE REVIEW

File: Example_for_code_review.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+this is just a simple text file for us to learn abput pull request - or as we prefer to short thier name - PR
+
+so
+
+lest say we have here a segment of code:
+
+
+nirs_method(int iterations){
+	for(int i=0 ; i < iterations ; i++) {
+		x=5;
```
Comment (TalShalmon) [inline Example_for_code_review.txt:13]: initilize x

File: Example_for_code_review.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+this is just a simple text file for us to learn abput pull request - or as we prefer to short thier name - PR
+
+so
+
+lest say we have here a segment of code:
+
+
+nirs_method(int iterations){
+	for(int i=0 ; i < iterations ; i++) {
```
Comment (TalShalmon) [inline Example_for_code_review.txt:12]: initilize x

File: Example_for_code_review.txt (hunk 1 above)
Comment (TalShalmon) [inline Example_for_code_review.txt:13]: x = 30

File: Example_for_code_review.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+		x=5;
+		x*=6;
```
Comment (TalShalmon) [inline Example_for_code_review.txt:14]: output 30

File: Nir/Example_for_code_review.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Hi!
+
+this is just a simple text file for us to learn abput pull request - or as we prefer to short thier name - PR
+
+so
+
+lest say we have here a segment of code:
+
+
+nirs_method(int iterations){
```
Comment (shalmonir) [inline Nir/Example_for_code_review.txt:11]: Hi @TalShalmon !
look at this comment and try to add some of your own

**Review by TalShalmon — APPROVED** [review]
My first CR

File: Example_for_code_review.txt (hunk 2 above)
Comment (shalmonir) [reply Example_for_code_review.txt:12]: right!

File: Example_for_code_review.txt (hunk 1 above)
Comment (shalmonir) [reply Example_for_code_review.txt:13]: cool, so, this is just a bit of a feel of how we reviewing. it's a conversation :)
after we done reviewing, if you would approve all my changes, Ill be able in PR amin screen to 'merge to master'. which mean, take all the commits I added here and put them in the main branch (master).
try create PR and I'll review

**Comment by shalmonir** [comment]
@TalShalmon 
if you got general comment, over this feature, you put it here (in case its not related to a line of code, but to logic or big picture implementation details)

**Pull request closed without merging**